### ISSUE TITLE
like button

### ISSUE THREAD

**Issue by Alexigbokwe** [kodersha/humhub-theme-flathub#18]
the like button is not working, can you help

**Comment by WebCrew** [comment]
At Sasha-Travkina`s theme the like button was always working, I wonder why it is no longer working. Maybe some JS files needs to become a update or so.

Maybe https://github.com/Felli can help you - she has done the latest updates of the FlatHub theme and created latest Pull Requests ;)

**Comment by Felli** [comment]
Hmm... I don't see any issues with the like button on my end, could you post what the Admin logs show for you?

**Comment by WebCrew** [comment]
Yeah same for me - as @Felli wrote, I also have no issues with the like button.
